js/src/wp-seo-term-scraper.js

```diff
@@ -76,6 +73,63 @@ window.yoastHideMarkers = true;
 		descriptionLabel.replaceWith( descriptionLabel.html() );
 	};
 
+	/**
+	 * Dispatches an action to the store that updates the snippet editor.
+	 *
+	 * @param {Object} data The data from the legacy snippet editor.
+	 *
+	 * @returns {void}
+	 */
+	const dispatchUpdateSnippetEditor = function( data ) {
+		/*
+		 * The setTimeout makes sure the React component is only rendered on the next
```

Comment:
Please explain why it's necessary for the component is updated on the next frame.

Also "rendered" should be "updated".